src/Filters/FiltersPartial.php

```diff
@@ -11,11 +12,20 @@ public function __invoke(Builder $query, $value, string $property) : Builder
         if (is_array($value)) {
             return $query->where(function (Builder $query) use ($value, $property) {
                 foreach ($value as $partialValue) {
-                    $query->orWhere($property, 'LIKE', "%{$partialValue}%");
+                    $query
+                        ->orWhere($property, 'LIKE', DB::raw('? ESCAPE "\"'))
+                        ->addBinding("%{$this->escapeLike($partialValue)}%");
                 }
             });
         }
 
-        return $query->where($property, 'LIKE', "%{$value}%");
+        return $query
+            ->where($property, 'LIKE', DB::raw('? ESCAPE "\"'))
+            ->addBinding("%{$this->escapeLike($value)}%");
+    }
+
+    private function escapeLike(string $value): string
```

Comment:
Can we make this one `protected`? This way it's a bit easier for people to extend if they'd want to.

Reply:
Of course!